src/Ops.Data/Ops/ClaimGroupRepository.cs

```diff
@@ -19,7 +19,7 @@ public class ClaimGroupRepository
 
         public async Task<bool> IsClaimGroup(string claim, string group)
         {
-            if(string.IsNullOrEmpty(group) || string.IsNullOrEmpty(claim))
+            if (string.IsNullOrEmpty(group) || string.IsNullOrEmpty(claim))
```

Comment:
I think this is more understandable and works the same?

`return !string.IsNullOrEmpty(group) && !string.IsNullOrEmpty(claim);`

Reply:
I think we need to keep it for the condition so, line 27 could execute if needed. Unless there was an alternative way for those conditions.